src/app/features/themeSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface ThemeState {
```

Comment:
prefixing interfaces with I would be better as that convention followed for other interfaces.